src/sidebar/store/modules/real-time-updates.ts

```diff
@@ -213,6 +213,14 @@ function hasPendingUpdates(state: State): boolean {
   return Object.keys(state.pendingUpdates).length > 0;
 }
 
+/**
+ * Return true if at least one annotation has been created or deleted on the
+ * server, but it has not yet been applied.
+ */
+function hasPendingUpdatesOrDeletions(state: State): boolean {
+  return pendingUpdateCount(state) > 0;
+}
```

Comment:
This function's name deviates from `pendingUpdateCount`, which actually counts both updates and deletions.

I considered changing the existing `hasPendingUpdates` one, but we also have a separated `hasPendingDeletion` selector which checks if a single annotation has been deleted.

In the end, I decided to create a new selector to make sure no regressions were introduced, but we might want to simplify the possible combinations at some point. Maybe after fully replacing the "updates button" with the "updates notification".